src/ts/components/subtitlesettings/backgroundcolorselectbox.ts

```diff
@@ -0,0 +1,48 @@
+import {SelectBox} from '../selectbox';
+import {SubtitleSettingConfig} from './subtitlesetting';
+import {UIInstanceManager} from '../../uimanager';
+import {SubtitleOverlay} from '../subtitleoverlay';
+import {ColorUtils, StorageUtils} from '../../utils';
+
+/**
+ * A select box providing a selection of different background colors.
+ */
+export class BackgroundColorSelectBox extends SelectBox {
+
+  private overlay: SubtitleOverlay;
```

Comment:
This field and the `constructor` are similar in all of your select boxes. Please add an `abstract class SubtitleSettingSelectBox exnteds SelectBox` and use it as a common base class for all your select boxes!